BetterStats/BetterStats.cs

```diff
@@ -665,7 +678,17 @@ public static void AddPlanetFactoryData(PlanetFactory planetFactory)
                 if (assembler.id != i || assembler.recipeId == 0) continue;
 
                 var frequency = 60f / (float)((double)assembler.timeSpend / 600000.0);
-                var speed = (float)(0.0001 * (double)assembler.speed);
+                var speed = (float)(0.0001 * Math.Max(assembler.speedOverride, assembler.speed));
+
+                // forceAccMode is true when Production Speedup is selected
+                if (assembler.forceAccMode)
+                {
+                    speed += speed * maxSpeedIncrease;
+                }
+                else
+                {
+                    frequency += frequency * maxProductivityIncrease;
```

Comment:
not sure about this as in productivity mode you just produce more while consuming the same amount of resources.

This means that in productivity mode we should use a multiplier that only applies to the productions of an assembler and not to it's consumptions

Reply:
You're right, I had that completely wrong. Updated now.